lib/scrabble.rb

```diff
@@ -1,7 +1,33 @@
+require 'pry'
 class Scrabble
 
   def score(word)
-    1
+    word = nil_value(word)
+    letters = word.upcase.chars
+    scoring = letters.map do |letter|
+      point_values[letter]
+    end
+    scoring.sum
```

Comment:
The above five lines can be compressed, better, and avoid using so many local variables.